client/settings/fraud-protection/advanced-settings/test/rule-toggle.test.tsx

```diff
@@ -155,29 +146,4 @@ describe( 'Fraud protection rule toggle tests', () => {
 			mockContext.protectionSettingsUI.test_rule.enabled
 		).toBeFalsy();
 	} );
-	test( 'sets the value correctly when block is selected', () => {
```

Comment:
Rather skip these tests so we can reuse them later.

Reply:
Fixed on [89e8fb8](https://github.com/Automattic/woocommerce-payments/pull/6275/commits/89e8fb8a7ca1bf8b090c0627a290fa60c973135b).